

vector_field:
    vr: 0.5 # reference forward speed for the vector field
    kf: 5.0 # convergence gain of the vector field
    # is_forward_motion: True # flag to invert the motion of the espeleorobo (move backwards or forward)
    reverse_direction: False # flag to invert the direction the curve will be followed

topics:
    pose_topic_name: "integrator/pose" # name of the topic in which the pose will be obtained
    pose_topic_type: "Pose" # Options: 'TFMessage', 'Pose' or 'Odometry'
    cmd_vel_topic_name: "integrator/vel" # name of the topic in which the linear velocities will be published
    path_topic_name: "example_path" # name of the topic in which the path will be published
    path_equation_topic_name: "example_path_equation" # name of the topic in which the path will be published (equation form)

#Collision avoidance considering the closest point in the world frame - works in 3D
obstacle_avoidance:
    flag_follow_obstacle: True # flag to enable the robot to follow an obstacle when it s blocking the vector field
    epsilon: 0.25 # reference distance between the robot and the path being followed
    switch_dist_0: 0.75 # distance from which the robot will start to follow the obstacle
    switch_dist: 0.5 # distance from which the robot will start to follow the obstacle
    obstacle_point_topic_name: "integrator/closest_point" # name of the topic in which the laser data will be obtained
